docs/security.md

```diff
@@ -23,7 +23,8 @@ If your applications are using event logging, the directory where the event logs
 
 ## Encryption
 
-Spark supports SSL for Akka and HTTP (for broadcast and file server) protocols. However SSL is not supported yet for WebUI and block transfer service.
+Spark supports SSL for Akka and HTTP (for broadcast and file server) protocols. SASL encryption is
+supported for the block transfer service. Encryption is not yet supported for the WebUI.
```

Comment:
Should we also explicitly say that encryption is not yet supported for shuffle data at rest (e.g. in spill files, shuffle files, data cached in memory or on-disk, etc)?